__init__.py

```diff
@@ -76,10 +78,29 @@ def _setup(self, force=False):
                     self.enable_fallback = \
                         self.settings.get('enable_fallback')
 
+                # Register tracker entities
+                self._list_tracker_entities()
+
     def _force_setup(self):
         self.log.debug('Creating a new HomeAssistant-Client')
         self._setup(True)
 
+    def _list_tracker_entities(self):
+        """List tracker entities.
+
+        Add them to entity file and registry it so
+        Padatious react only to known entities.
+        Should fix conflict with Where is skill.
+        """
+        types = ['device_tracker']
```

Comment:
I presume this list might be extended to include other types in the future?

Reply:
Yes, it can be extended, but I do not recommend it now. I tried to register all entities that this skill can handle in my HA instance, well it was a bad idea. Padatious simply stopped responding.

Reply:
How many is typical? 

There is a timeout on training for Padatious intents (each intent is a small neural network) so I presume that's what got hit. I don't think there is a way to temporarily extend that at the moment, but it's certainly something that is possible if it is found to be the issue.

Reply:
For now, we can live with just tracker. If there is future conflict we can check it. For now I have about 4 trackers and they run ok.